mcs/class/System/System.Net/MacProxy.cs

```diff
@@ -326,15 +391,144 @@ public override string ToString ()
 		{
 			return Create (str);
 		}
+
+		// to be used when an API like CF*Get* returns a CFString
+		internal static string FetchString (IntPtr handle)
```

Comment:
There's already `AsString` which looks like it does exactly the same as `FetchString`.